seatunnel-formats/seatunnel-format-json/src/test/java/org/apache/seatunnel/format/json/canal/CanalJsonSerDeSchemaTest.java

```diff
@@ -69,6 +74,105 @@ public void testDeserializeNullRow() throws Exception {
         assertEquals(0, collector.list.size());
     }
 
+    @Test
+    public void testDeserializeNoJson() throws Exception {
+        final CanalJsonDeserializationSchema deserializationSchema =
+                createCanalJsonDeserializationSchema(null, null);
+        final SimpleCollector collector = new SimpleCollector();
+        String noJsonMsg = "{]";
+        boolean isExcept = false;
+
+        SeaTunnelRuntimeException expected = CommonError.jsonOperationError(FORMAT, noJsonMsg);
+        try {
+            deserializationSchema.deserialize(noJsonMsg.getBytes(), collector);
+        } catch (Throwable t) {
+            isExcept = true;
+            assertEquals(t.getClass(), IOException.class);
+            Throwable cause = t.getCause();
+            assertSame(cause.getClass(), expected.getClass());
+            assertEquals(cause.getMessage(), expected.getMessage());
+        }
```

Comment:
You can use `Assertions.assertThrows` to catch exception. Please refer https://github.com/apache/seatunnel/blob/af2acb16fe315f670c5909f7489042ed1c9b191d/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-7/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcErrorIT.java#L155-L169